test/test_dynamic_type.cpp

```diff
@@ -1188,11 +1188,21 @@ TEST_F(DynamicTypeTest, MemberPointer) {
   static_assert(b->*&B::x == 3);
   static_assert(b->*&B::y == 4);
   constexpr ABCD c = CD{C{5, 6}};
+#if __cplusplus >= 202002L
```

Comment:
Unfortunately, this can no longer be constexpr in C++17. In C++20, it should be constexpr, but I have not tested it (because there are other failures on C++20 that I need to look into).